lib/config/definitions.ts

```diff
@@ -1999,6 +1999,13 @@ const options: RenovateOptions[] = [
     type: 'boolean',
     default: false,
   },
+  {
+    name: 'allowMergeWhenPipelineSucceeds',
+    description: `Enable or disable usage of GitLab's "merge when pipeline succeeds" feature when automerging PRs`,
+    type: 'boolean',
+    default: false,
+    stage: 'global',
```

Comment:
Are you certain that we don't want this configurable per-repo?

Reply:
I think yes, for the reason described in https://github.com/renovatebot/renovate/issues/3265#issuecomment-536226275. Would you as the owner of the Renovate app on GitLab.com want to get complaints when people activate this in their repos and it fails?

Reply:
I usually don’t mind if people opt into a feature they don’t understand and it doesn’t work as expected. But it depends on the failure case. I think if the feature is opt in and the risk is documented then I’m pretty unlikely to get complaints, maybe just a question or two.

Reply:
Alright, I’ll change it so that it can also be configured from within the repo!

Reply:
Changed with ea783946d0d76f953c6fe3b4c3f21026d2318010